Assets/Samples/CustomComposite/CustomComposite.cs

```diff
@@ -136,13 +136,6 @@ public class CustomCompositeEditor : InputParameterEditor<CustomComposite>
 {
     public override void OnGUI()
     {
-        // Using the 'target' property, we can access an instance of our composite.
-        var currentValue = target.scaleFactor;
-
-        // The easiest way to lay out our UI is to simply use EditorGUILayout.
-        // We simply assign the changed value back to the 'target' object. The input
-        // system will automatically detect a change in value.
-        target.scaleFactor = EditorGUILayout.Slider(m_ScaleFactorLabel, currentValue, 0, 2);
     }
```

Comment:
OnGUI is public, unfortunately. So we can't kill those in this PR without releasing a new major version.

Reply:
Just makes it quicker to review/look at since it activates less brain cells when looking at the same code 2 years later :)

Reply:
Leave it then, not a big deal